project/cmake/README.md

```diff
@@ -168,13 +221,36 @@ scanning them would take up an unreasonable amount of configure time.
 When using the makefile builds a few extra targets are defined:
 
 - `make check` builds and executes the test suite.
+- `make check-valgrind` builds and executes the test suite with valgrind memcheck.
 
 Code coverage (with Gcov, LCOV and Gcovr) can be built on Linux:
 
 - CMake has to be executed with `-DCMAKE_BUILD_TYPE=Coverage`
 - `make coverage` generates an HTML code coverage report.
 - `make coverage_xml` generates an XML code coverage report.
 
+## Building binary addons
```

Comment:
My main usecase when adding the `make binary-addons` was testing packaging for osx/ios/android. I thought it would be useful also for users that build kodi, but don't develop addons.
I'll make this more clear in the documentation.

Since I didn't yet have played much with the binary-addon build system, maybe you, @wsnipex, @hudokkow, @garbear or @AchimTuran can help documenting their workflow. That would also make it more clear to me what issues you currently face when working on the addons.